crates/vault-registry/src/lib.rs

```diff
@@ -417,6 +417,21 @@ pub mod pallet {
             Self::_set_liquidation_collateral_threshold(currency_pair, threshold);
             Ok(())
         }
+
+        /// Changes the collateral liquidation threshold for a currency (only executable by the Root account)
+        ///
+        /// # Arguments
+        /// * `version` - the semver version, where the zero-index element is the major version
+        /// * `checksum` - the SHA256 checksum of the client binary
+        #[pallet::weight(<T as Config>::WeightInfo::set_liquidation_collateral_threshold())]
+        #[transactional]
+        pub fn set_client_release(origin: OriginFor<T>, version: [u32; 3], checksum: [u8; 32]) -> DispatchResult {
+            ensure_root(origin)?;
+            // TODO: Should the new semver version have a check that ensures it is striclty ascending?
```

Comment:
Yes, that would make sense. However, I'm wondering if semver versions are enough. We might have hotfixes that fall outside of semver.

Reply:
Resolving as we agreed on storing the release URL instead of the version